CareerCatalyst/Career-Catalyst
# 49 delete descriptions off cards

## Description
Removed the extra content on the homage cards, and by that I mean I did not delete the descriptions off of the cards just took of the slot in the feature component so that the descriptions can still be used elsewhere. Also I changed the background color of the index page since i realized it was wrong.


## Related Issue

#49 

## Acceptance Criteria

[Delete descriptions off cards](https://github.com/CareerCatalyst/Career-Catalyst/issues/49)

 - [x] This is the first criteria
 - [x] Second criteria
 - [x] Third criteria

## Type of Changes

Use a check for  ✓ the following type of changes: 

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓   | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates
`src/components/ContentMedia.astro`: Removed the extra unnecessary content in the main page
`src/assets/scss/base/_font.scss`: changed the hover styling
`src/components/Feature.astro` | `src/pages/index.astro`: Took off the descriptions out of the cards in the the index file as to lessen decision overload

### Before
![229367329-7dcb1457-80fa-4e6a-b19c-e5aa213166d8](https://user-images.githubusercontent.com/20276785/232618967-ac175997-1bc1-4634-b398-feccca01d286.png)


### After

![Screenshot 2023-04-17 174126](https://user-images.githubusercontent.com/20276785/232618995-811c919c-db1f-4b80-8b8e-81119ab86fcf.png)


## Testing Steps / QA Criteria

The hover animation is currently not working as well as it should so we have to make that an issue to properly set them.
